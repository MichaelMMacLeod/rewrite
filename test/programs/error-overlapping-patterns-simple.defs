(def A Y)
(def A Z)