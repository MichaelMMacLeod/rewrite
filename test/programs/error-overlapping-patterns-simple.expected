error[E007]: overlapping patterns
./test/programs/error-overlapping-patterns-simple.defs:1:6
   |
 1 | (def A Y)
   |      ^ this pattern may match the same term as ...
./test/programs/error-overlapping-patterns-simple.defs:2:6
   |
 2 | (def A Z)
   |      ^ ... this other pattern
help: patterns possibly matching the same term are not allowed
